plugins/plugin-chart-echarts/src/BoxPlot/controlPanel.ts

```diff
@@ -64,6 +64,97 @@ export default {
       expanded: true,
       controlSetRows: [
         ['color_scheme'],
+        [
+          {
```

Comment:
Could we define these controls to `superset-ui/chart-controls`? (We should reuse same control)